batch/batch/driver/main.py

```diff
@@ -393,140 +404,162 @@ def validate_int(name, value, predicate, description):
         set_message(session,
                     'Either the worker must use a local SSD or PD SSD data disk must be non-zero.',
                     'error')
-        raise web.HTTPFound(deploy_config.external_url('batch-driver', '/'))
+        raise web.HTTPFound(deploy_config.external_url('batch-driver', pool_url_path))
+
     if worker_local_ssd_data_disk and worker_pd_ssd_data_disk_size_gb > 0:
         set_message(session,
                     'Worker cannot both use local SSD and have a non-zero PD SSD data disk.',
                     'error')
-        raise web.HTTPFound(deploy_config.external_url('batch-driver', '/'))
+        raise web.HTTPFound(deploy_config.external_url('batch-driver', pool_url_path))
 
     max_instances = validate_int(
         'Max instances',
         post['max_instances'],
         lambda v: v > 0,
         'a positive integer')
 
-    pool_size = validate_int(
-        'Worker pool size',
-        post['pool_size'],
+    max_live_instances = validate_int(
+        'Max live instances',
+        post['max_live_instances'],
         lambda v: v > 0,
         'a positive integer')
 
     enable_standing_worker = 'enable_standing_worker' in post
 
-    await inst_pool.configure(
-        worker_type, worker_cores, worker_disk_size_gb,
+    await pool.configure(
+        worker_cores, boot_disk_size_gb,
         worker_local_ssd_data_disk, worker_pd_ssd_data_disk_size_gb,
-        standing_worker_cores,
-        max_instances, pool_size, enable_standing_worker)
+        enable_standing_worker, standing_worker_cores,
+        max_instances, max_live_instances)
 
     set_message(session,
-                'Updated batch configuration.',
+                f'Updated configuration for {pool}.',
                 'info')
 
-    return web.HTTPFound(deploy_config.external_url('batch-driver', '/'))
+    return web.HTTPFound(deploy_config.external_url('batch-driver', pool_url_path))
 
 
-@routes.get('/user_resources')
+@routes.get('/inst_coll/{inst_coll}')
```

Comment:
I changed this route name. For now, it's always a pool, but may not be in the future. In future PRs, I'll make this function render the correct page depending on the type of the Instance collection.